backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
Assuming it is only supposed to be set once, a common pattern for cases like this is to have the var be private, and then expose a function which will complain if one tries to overwrite it when the value is Some. There's not really a totally clean solution to the problem, but sometimes you can't avoid being in this situation.